tests/crypto/test_cases/test_aead.c

```diff
@@ -193,7 +193,7 @@ void aead_chachapoly_setup_simple(void)
 	unhexify_aead();
 }
 
-#if defined(CONFIG_MBEDTLS_CCM_C)
+#if defined(CONFIG_CC310_MBEDTLS_CCM_C) || defined(CONFIG_VANILLA_MBEDTLS_CCM_C) || defined(CONFIG_MBEDTLS_CCM_C)
```

Comment:
Since we are in source code, it would suffice with MBEDTLS_CCM_C (defined in nrf-config.h)

Reply:
Removed in 8abda7e09d0c871e53353e762735ecb9426cd792